bin/get_pending_task.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import time
```

Comment:
do we need to check these in? they look like debugging scripts

if needed -- please put them with the taskworker code so they don't go unowned / unused and please add appropriate tests